static/app/components/events/interfaces/stackTraceV2.tsx

```diff
@@ -36,8 +36,13 @@ function StackTrace({
   hasHierarchicalGrouping,
   groupingCurrentLevel,
 }: Props) {
+  function getPlatform(): PlatformType {
+    const framePlatform = data.frames?.find(frame => !!frame.filename);
+    return framePlatform?.platform ?? event.platform ?? 'other';
```

Comment:
Does each frame potentially define a platform now?

Reply:
here I am using a similar logic that we already use in this file https://github.com/getsentry/sentry/blob/0ad534f26b074d60307d896df03f02ec0ccf5efb/static/app/components/events/interfaces/frame/line.tsx#L347

we assume that every frame have the same platform but it can also happen that none of them have a platform and if this happens it falls back to the event event platform and if this is also undefined than the function return other

Reply:
in the logic above there is a typo... it shall not be filename but platform... let me fix it